Wrappers/Python/cil/utilities/dataexample.py

```diff
@@ -87,54 +91,60 @@ class TestData(object):
     RESOLUTION_CHART = 'resolution_chart.tiff'
     SIMPLE_PHANTOM_2D = 'hotdog'
     SHAPES =  'shapes.png'
-    
+    RAINBOW = 'rainbow.png'
     '''
     BOAT = 'boat.tiff'
     CAMERA = 'camera.png'
     PEPPERS = 'peppers.tiff'
     RESOLUTION_CHART = 'resolution_chart.tiff'
     SIMPLE_PHANTOM_2D = 'hotdog'
     SHAPES =  'shapes.png'
-
+    RAINBOW =  'rainbow.png'
     
     def __init__(self, **kwargs):
         self.data_dir = kwargs.get('data_dir', data_dir)
         
     def load(self, which, size=(512,512), scale=(0,1), **kwargs):
         if which not in [TestData.BOAT, TestData.CAMERA, 
                          TestData.PEPPERS, TestData.RESOLUTION_CHART,
-                         TestData.SIMPLE_PHANTOM_2D, TestData.SHAPES]:
+                         TestData.SIMPLE_PHANTOM_2D, TestData.SHAPES,
+                         TestData.RAINBOW]:
             raise ValueError('Unknown TestData {}.'.format(which))
         if which == TestData.SIMPLE_PHANTOM_2D:
             N = size[0]
             M = size[1]
             sdata = numpy.zeros((N, M))
             sdata[int(round(N/4)):int(round(3*N/4)), int(round(N/4)):int(round(3*N/4))] = 0.5
             sdata[int(round(M/8)):int(round(7*M/8)), int(round(3*M/8)):int(round(5*M/8))] = 1
-            ig = ImageGeometry(voxel_num_x = N, voxel_num_y = M, dimension_labels=[ImageGeometry.HORIZONTAL_X, ImageGeometry.HORIZONTAL_Y])
+            ig = ImageGeometry(voxel_num_x = M, voxel_num_y = N, dimension_labels=[ImageGeometry.HORIZONTAL_Y, ImageGeometry.HORIZONTAL_X])
             data = ig.allocate()
             data.fill(sdata)
-            
+
         elif which == TestData.SHAPES:
             with Image.open(os.path.join(self.data_dir, which)) as f:
                 tmp = numpy.array(f.convert('L'))
             N = 200
             M = 300   
-            ig = ImageGeometry(voxel_num_x = N, voxel_num_y = M, dimension_labels=[ImageGeometry.HORIZONTAL_X, ImageGeometry.HORIZONTAL_Y])
+            ig = ImageGeometry(voxel_num_x = M, voxel_num_y = N, dimension_labels=[ImageGeometry.HORIZONTAL_Y, ImageGeometry.HORIZONTAL_X])
             data = ig.allocate()
             data.fill(tmp/numpy.max(tmp))
             
         else:
             with Image.open(os.path.join(self.data_dir, which)) as tmp:
                 bands = tmp.getbands()
```

Comment:
For the Rainbow, we need `convert('RGB')` at the end

Reply:
np.array(Image.open(path_to_image + "rainbow.png").convert('RGB')

Reply:
Also the size should not be the [default](https://github.com/vais-ral/CCPi-Framework/pull/802/files#diff-68a90db86e1c91b0e20f919b6c805a7f2f5e5de02bdf4fb775103b744245c053R107) `(512,512)`

Reply:
All updated. And shapes fixed to resize properly.